src/main/resources/data/batch-voucher-export-configs/credentials/test_config2_credentials.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think "id" could be specified for credential samples. Just to be aligned with other sample data

Reply:
done